### ISSUE TITLE
Password does not end up in clipboard

### ISSUE THREAD

**Issue by w2ak** [CGenie/alfred-pass#9]
When using alfred, completion works fine and I can find my password. Then I hit enter and "nothing" happens, i.e., my password is not copied to clipboard.
I read the script `pass-show.sh`, when I use it in my terminal it works fine and copies to clipboard but not when it is executed through alfred.

I'm using iTerm2 with zsh but executing pass-show.sh with /bin/bash, which is specified in the script's shebang.

Right now I did not find what was the problem. If you have any idea I'll look into making the workflow compatible with my situation.

**Comment by JiDai** [comment]
I am in the same situation (Iterm2/ZSH) and I use Alfred 3. Did you find a solution ?

**Comment by w2ak** [comment]
Do you mean apart from switching to Linux ? :P

**Comment by JiDai** [comment]
Yes :)

**Comment by CGenie** [comment]
Hello,
Sorry for large delay, I was busy and pass didn't work for a long time in my Mac, recently, because I made some changes and wasn't quick to fix them :)
Anyways I think the problem lies somewhere with GPG -- when I hit Enter on the password I'm not asked for GPG password. But when I ran the script from console, i did get an answer. I remember that GPG thing to be a bit tricky, maybe will find what is the issue.

**Comment by CGenie** [comment]
Yeah, in Alfred workflow you can turn on debug mode. This is what I get:

```

[2017-09-30 22:57:29][ERROR: action.script] pass-show.sh: line 11: HD/Users/pk/.gnupg/S.gpg-agent:712:1: No such file or directory
```

**Comment by CGenie** [comment]
https://github.com/CGenie/alfred-pass/tree/turn-gpg-off

I turned off GPG. Can you check if it works now? (need to `make` this and upload file to Alfred).

**Comment by JiDai** [comment]
In alfred debug console, I got this after select an item : 

```
[2017-10-01 09:39:59][ERROR: action.script] gpg-agent[28909]: Attention : << --write-env-file >> est une option obsol`ete - non prise en compte
gpg-agent: une instance de gpg-agent fonctionne déjà —
pas de démarrage d'une nouvelle instance
gpg: 'echec du d'echiffrement : No secret key
There is no password to put on the clipboard at line 1.
```

EDIT : It is finally working, I launched the command in terminal, got the invite passphrase and after validate it, alfred workflow copied my passwords

**Comment by CGenie** [comment]
Could you please try the code with gpg stuff commented out? It seems to
work for me.

01.10.2017 8:51 AM "Jordi Dosne" <notifications@github.com> napisał(a):

**Comment by bgschiller** [comment]
For anyone else having this issue, you should be able to change your pinentry program to `pinentry-mac` so that you're still prompted for your passphrase. I give more details in this blog post: https://brianschiller.com/blog/2016/08/31/gnu-pass-alfred, and in #14
